docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -82,7 +82,42 @@ Disable bootstrapping enrollment by setting `ZITI_BOOTSTRAP_ENROLLMENT=false` in
 
 ## Firewall
 
-The router's generated configuration always listens on a single, configurable TCP port (default: `3022`). You may set `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env` to bootstrap with a different port.
+The router's generated configuration always uses a single TCP port (default: `3022`) defined by `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env`.
```

Comment:
would you remove the word 'always'? it's an extra word that doesn't add any additional clarity.